### DISCUSSION TITLE
[newbie] How to install in Arduino IDE?

### DISCUSSION THREAD

**Discussion by Shohreh** [Bodmer/TFT_eSPI#1566] [Q&A - General] [answered]
Hello,

This is the first time I'm using the Arduino IDE (1.8.19 on Windows).

I need to compile a small program for the TTGO micro-controller that requires the TFT_eSPI library.

Since it's installed by default (2.4.2), I did nothing more and simple compiled the program… which fails with "fatal error: TFT_eSPI.h: No such file or directory"

[The page in the wiki](https://github.com/Bodmer/TFT_eSPI/wiki) about installing in the Arduino IDE is empty.

Do I need to install additional files? Do I also need to change the board for which it compiles (At the bottom right, the status bar says "Arduino Uno")?

Thank you.

**Comment by Bodmer** [comment, marked as answer]
The library is not installed by default, you have to load it via the Arduino IDE library manager:  menu Sketch->Include Library->Manage libraries
Put TFT_eSPI in the search box and load it.

It will not run on an UNO, you need to select the board you are using.

**Reply by Shohreh** [reply]
Thank you.

I (obviously) assumed having the library installed in the IDE and having an "include" line in the source file was enough to use a library.

After installing an ESP32 set of boards, and upgrading that library to the latest and greatest through Sketch > Include Library > Manage  Libraries > Install,  compilng went fine.
